v0/destinations/marketo/transform.js

```diff
@@ -294,6 +307,10 @@ const getLeadId = async (message, formattedDestination, token) => {
 // Almost same as leadId lookup. Noticable difference from lookup is we'll using
 // `id` i.e. leadId as lookupField at the end of it
 const processIdentify = async (message, formattedDestination, token) => {
+  // If mapped to destination, Add externalId to traits
+  if (get(message, MappedToDestinationKey)) {
```

Comment:
[Non-blocking] It'd be good if you can abstract this into a util function (say `isEventMappedToDestination` or `isVDMEvent`).

Reply:
Will raise another pr for this to make the change for every destination. Will work?

Reply:
Yes. I mentioned that it's a non-blocking comment.